packages/modules/src/modules/epics/ContributionsEpicButtons.tsx

```diff
@@ -174,7 +176,12 @@ const PrimaryCtaButtonApplePay = ({
 
     return (
         <div css={buttonMarginStyles(true)}>
-            <ButtonApplePay onClickAction={urlWithRegionAndTracking}>{buttonText}</ButtonApplePay>
+            <ButtonApplePay
+                submitComponentEvent={submitComponentEvent}
```

Comment:
Yep OphanEvent now being routed down into ApplePay button, have checked in CODE this fires upon pressing.